### ISSUE TITLE
Merge bug list error

### ISSUE THREAD

**Issue by grahnavard** [biobakery/MetaPhlAn#154]
**Describe the bug**
The utility function produces the following error:
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pandas/io/parsers.py", line 1947, in __init__
    _validate_usecols_names(usecols, self.names)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pandas/io/parsers.py", line 1232, in _validate_usecols_names
    raise ValueError(
ValueError: Usecols do not match columns, columns expected but not found: [0, 1, 2]

**Platform (please complete the following information):**
 - Version 3 and python 3
 - Download source : metaphlan github

**Additional context**
Metahplan2 works with no error. I think version 3 assumes three columns in the output.

**Comment by fbeghini** [comment]
Hi @grahnavard ,
Can you upload the files you are trying to merge?

**Comment by DooYal** [comment]
Similarily, I can successfully merge bug list by "MetaPhlAn version 3.0.14 (19 Jan 2022) with humann3" , however, it would be failed in the process of building the heatmap. I tried to run the same analysis on humann2 and metaplan2, it worked.
I found the critical difference between v2 and v3 is the bug list, 
Here is a sceenshot of the bug list by v3 software
![image](https://user-images.githubusercontent.com/31724229/174545033-5282c9e1-bdd4-4171-b988-4518277f7c50.png)
![image](https://user-images.githubusercontent.com/31724229/174546446-5aa047ce-9e8c-40bb-b763-063fff3be5d5.png)
There is a column called "additional_species" with only a few raws listing some records here, however, there is no such column in the bug list generated by v2 software, so I guess these records processed by the v3 software may affect the following process.
